src/RageSoundReader_ThreadedBuffer.cpp

```diff
@@ -237,7 +236,9 @@ void RageSoundReader_ThreadedBuffer::BufferingThread()
 		else
 		{
 			m_Event.Unlock();
-			usleep( static_cast<int>((fTimeToSleep * 1000000) + 0.5) );
+			const auto sleep_duration = std::chrono::duration_cast<std::chrono::microseconds>(
```

Comment:
I think you can do:

```cpp
std::chrono::microseconds{std::chrono::duration<float>(fTimeToSleep)}; 
```

Which is slighttlllyy more succinct.

ALSO we should suffix the variable with units (is it ms, is it secs, etc.) It makes it more clearer.